src/Tizen.NUI/src/public/BaseComponents/TextMapHelper.cs

```diff
@@ -530,14 +529,14 @@ public static Placeholder GetPlaceholderStruct(PropertyMap map)
             PropertyValue pixelSizeValue = null;
             bool ellipsis = false;
 
-            map.Find(0)?.Get(out text);
-            map.Find(1)?.Get(out textFocused);
-            map.Find(2)?.Get(color);
-            map.Find(3)?.Get(out fontFamily);
-            map.Find(4)?.Get(fontStyle);
-            pointSizeValue = map.Find(5);
-            pixelSizeValue = map.Find(6);
-            map.Find(7)?.Get(out ellipsis);
+            map.Find(0, "text")?.Get(out text);
```

Comment:
Could you please share an example of the problem?

The placeholder's PropertyMap always has index 0.
So, in the case of this, the property value of index 0 is always returned.

Reply:
Hello, 
Same as above, please refer the case: "TextMapHelperGetPlaceholderStruct" ,
and the path is: TizenFX\test\Tizen.NUI.Tests\Tizen.NUI.Devel.Tests\testcase\public\BaseComponents\TSTextMapHelper.cs

In this case, we will got failed, the out string is empty too.

Reply:
Hello @AchoWang 
Thanks for reporting the problem.
I fixed the problematic tc.
Please refer PR.
https://github.com/Samsung/TizenFX/pull/3815

Reply:
Hi @wonrst 
I notice some functions  in TextMapHelper.cs  like GetUnderlineStruct, GetShadowStruct, GetOutlineStruct and so on.
Is it better to consider unification?
Or what's difference between them?

Reply:
This depends on how the DALi toolkit's text component returns the PropertyMap.
Some are index base, some are string key base.
So each property is different.

It would be nice to unify for consistency, 
but this can make to different behavior. And this can causes compatibility issues.
I think it is better to keep it as it is.

Reply:
OK, I got it!
Thanks very much!